roachtest: asyncpg failed

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4368530&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4368530&tab=artifacts#/asyncpg) on master @ [bbb473c8f304ac20fec51ff0a0d04e128383bcf6](https://github.com/cockroachdb/cockroach/commits/bbb473c8f304ac20fec51ff0a0d04e128383bcf6):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:191,orm_helpers.go:117,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1226-gbbb473c8f3
		Tests run against asyncpg v0.24.0
		134 Total Tests Run
		126 tests passed
		8 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_transaction.TestTransaction.test_transaction_interface_errors (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
		--- FAIL: test_timeout.TestTimeoutCoversPrepare.test_timeout_covers_prepare_01 (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_nested_isolation_level (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_within_manual_transaction (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_isolation_level (unexpected)
		--- FAIL: test_timeout.TestTimeout.test_timeout_05 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-13147

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4378266&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4378266&tab=artifacts#/asyncpg) on master @ [9514f13d0190849eda12b39e272c278b82c269f4](https://github.com/cockroachdb/cockroach/commits/9514f13d0190849eda12b39e272c278b82c269f4):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:191,orm_helpers.go:117,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1272-g9514f13d01
		Tests run against asyncpg v0.24.0
		134 Total Tests Run
		126 tests passed
		8 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_transaction.TestTransaction.test_isolation_level (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_interface_errors (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_nested_isolation_level (unexpected)
		--- FAIL: test_timeout.TestTimeoutCoversPrepare.test_timeout_covers_prepare_01 (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
		--- FAIL: test_timeout.TestTimeout.test_timeout_05 (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_within_manual_transaction (unexpected)
		--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4445360&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4445360&tab=artifacts#/asyncpg) on master @ [a7449866f3229872d92e1de973e209838f07de35](https://github.com/cockroachdb/cockroach/commits/a7449866f3229872d92e1de973e209838f07de35):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1614-ga7449866f3
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4455314&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4455314&tab=artifacts#/asyncpg) on master @ [978744dbb94b03497154547ce7fa97ea3be9a401](https://github.com/cockroachdb/cockroach/commits/978744dbb94b03497154547ce7fa97ea3be9a401):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1656-g978744dbb9
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4464217&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4464217&tab=artifacts#/asyncpg) on master @ [8e7248296a2b8b2a4d07dd82678c687a02d56c4f](https://github.com/cockroachdb/cockroach/commits/8e7248296a2b8b2a4d07dd82678c687a02d56c4f):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1732-g8e7248296a
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4467588&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4467588&tab=artifacts#/asyncpg) on master @ [eee8ef080a95bc59eb3ddc2bebf623eb78e78db9](https://github.com/cockroachdb/cockroach/commits/eee8ef080a95bc59eb3ddc2bebf623eb78e78db9):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1740-geee8ef080a
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4470933&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4470933&tab=artifacts#/asyncpg) on master @ [9b45e990d5ddbbd6da13e8d3e312bb251fe0233a](https://github.com/cockroachdb/cockroach/commits/9b45e990d5ddbbd6da13e8d3e312bb251fe0233a):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1746-g9b45e990d5
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4479919&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4479919&tab=artifacts#/asyncpg) on master @ [d89e743edc44da85d9bf4e9087b451e11d3628bd](https://github.com/cockroachdb/cockroach/commits/d89e743edc44da85d9bf4e9087b451e11d3628bd):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1763-gd89e743edc
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4496763&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4496763&tab=artifacts#/asyncpg) on master @ [dbdbb93978a9ad564e07085c15807f415ccaa00a](https://github.com/cockroachdb/cockroach/commits/dbdbb93978a9ad564e07085c15807f415ccaa00a):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.1-1836-gdbdbb93978
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4505561&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4505561&tab=artifacts#/asyncpg) on master @ [0e1edf3ad9da29b1d3bb28fcfc0f293d700e3cb9](https://github.com/cockroachdb/cockroach/commits/0e1edf3ad9da29b1d3bb28fcfc0f293d700e3cb9):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.1-1878-g0e1edf3ad9
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4510241&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4510241&tab=artifacts#/asyncpg) on master @ [71eb44ffb1574350cf4a36143ed423f6ac0ca02b](https://github.com/cockroachdb/cockroach/commits/71eb44ffb1574350cf4a36143ed423f6ac0ca02b):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.1-1893-g71eb44ffb1
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4514489&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4514489&tab=artifacts#/asyncpg) on master @ [3b7f761f019c0457c19726bab8d14c231cd318fc](https://github.com/cockroachdb/cockroach/commits/3b7f761f019c0457c19726bab8d14c231cd318fc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.1-1911-g3b7f761f01
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4522718&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4522718&tab=artifacts#/asyncpg) on master @ [c6a5a395cbecbbd5ba77c2c14e3bf68eb59eeb19](https://github.com/cockroachdb/cockroach/commits/c6a5a395cbecbbd5ba77c2c14e3bf68eb59eeb19):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-186-gc6a5a395cb
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4531460&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4531460&tab=artifacts#/asyncpg) on master @ [e7e596ede9d3da877e313a3d39ab498a5cb7d27b](https://github.com/cockroachdb/cockroach/commits/e7e596ede9d3da877e313a3d39ab498a5cb7d27b):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-210-ge7e596ede9
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4539211&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4539211&tab=artifacts#/asyncpg) on master @ [5755663effbdbe8175bb74e5cdc19b2298e0a0bc](https://github.com/cockroachdb/cockroach/commits/5755663effbdbe8175bb74e5cdc19b2298e0a0bc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-251-g5755663eff
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4549874&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4549874&tab=artifacts#/asyncpg) on master @ [0069e980d756ca580167eb321ac6b348b35313f0](https://github.com/cockroachdb/cockroach/commits/0069e980d756ca580167eb321ac6b348b35313f0):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-315-g0069e980d7
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4558024&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4558024&tab=artifacts#/asyncpg) on master @ [270eb7b3ab88fc47ba86d87f4e2113e560f047a8](https://github.com/cockroachdb/cockroach/commits/270eb7b3ab88fc47ba86d87f4e2113e560f047a8):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-342-g270eb7b3ab
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4561635&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4561635&tab=artifacts#/asyncpg) on master @ [dc5432d65b9c38ce1ce5832e3c49d345b66145dc](https://github.com/cockroachdb/cockroach/commits/dc5432d65b9c38ce1ce5832e3c49d345b66145dc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-352-gdc5432d65b
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4565858&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4565858&tab=artifacts#/asyncpg) on master @ [231fd420c61e39b6ad08c982c496d82cf1910bc5](https://github.com/cockroachdb/cockroach/commits/231fd420c61e39b6ad08c982c496d82cf1910bc5):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.2-356-g231fd420c6
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4574775&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4574775&tab=artifacts#/asyncpg) on master @ [f5fc84fb5707428ae9505c5e3e90cf3f63d465ad](https://github.com/cockroachdb/cockroach/commits/f5fc84fb5707428ae9505c5e3e90cf3f63d465ad):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/asyncpg/run_1
	orm_helpers.go:190,orm_helpers.go:116,asyncpg.go:130,asyncpg.go:141,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.3-232-gf5fc84fb57
		Tests run against asyncpg v0.24.0
		132 Total Tests Run
		130 tests passed
		2 tests failed
		34 tests skipped
		96 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test_pool.TestPool.test_pool_11 (unexpected)
		--- FAIL: test_cursor.TestCursor.test_cursor_03 (unexpected)
		For a full summary look at the asyncpg artifacts 
		
		An updated blocklist (asyncpgBlocklist22_1) is available in the artifacts' asyncpg log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/asyncpg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/asyncpg_blocklist.go
@@ -23,11 +23,7 @@ var asyncpgBlocklists = blocklistsForVersion{
 // Please keep these lists alphabetized for easy diffing.
 // After a failed run, an updated version of this blocklist should be available
 // in the test log.
-var asyncpgBlocklist22_1 = asyncpgBlocklist21_2
-
-var asyncpgBlocklist21_2 = blocklist{}
-
-var asyncpgIgnoreList22_1 = blocklist{
+var asyncpgBlocklist22_1 = blocklist{
 	"test_cache_invalidation.TestCacheInvalidation.test_prepare_cache_invalidation_in_pool":               "unknown",
 	"test_cache_invalidation.TestCacheInvalidation.test_prepare_cache_invalidation_in_transaction":        "unknown",
 	"test_cache_invalidation.TestCacheInvalidation.test_prepare_cache_invalidation_silent":                "unknown",
@@ -37,7 +33,6 @@ var asyncpgIgnoreList22_1 = blocklist{
 	"test_cache_invalidation.TestCacheInvalidation.test_type_cache_invalidation_in_transaction":           "unknown",
 	"test_cache_invalidation.TestCacheInvalidation.test_type_cache_invalidation_on_change_attr":           "unknown",
 	"test_cache_invalidation.TestCacheInvalidation.test_type_cache_invalidation_on_drop_type_attr":        "unknown",
-	"test_cancellation.TestCancellation.test_cancellation_03":                                             "unknown",
 	"test_codecs.TestCodecs.test_array_with_custom_json_text_codec":                                       "unknown",
 	"test_codecs.TestCodecs.test_composites_in_arrays":                                                    "unknown",
 	"test_codecs.TestCodecs.test_enum":                                                                    "unknown",
@@ -72,7 +67,6 @@ var asyncpgIgnoreList22_1 = blocklist{
 	"test_cursor.TestIterableCursor.test_cursor_iterable_02":                                              "unknown",
 	"test_exceptions.TestExceptions.test_exceptions_str":                                                  "unknown",
 	"test_exceptions.TestExceptions.test_exceptions_unpacking":                                            "unknown",
-	"test_execute.TestExecuteMany.test_executemany_client_failure_after_writes":                           "unknown",
 	"test_execute.TestExecuteMany.test_executemany_client_failure_in_transaction":                         "unknown",
 	"test_execute.TestExecuteMany.test_executemany_client_server_failure_conflict":                        "unknown",
 	"test_execute.TestExecuteMany.test_executemany_prepare":                                               "unknown",
@@ -83,7 +77,6 @@ var asyncpgIgnoreList22_1 = blocklist{
 	"test_execute.TestExecuteMany.test_executemany_timeout_flow_control":                                  "unknown",
 	"test_execute.TestExecuteScript.test_execute_script_3":                                                "unknown",
 	"test_execute.TestExecuteScript.test_execute_script_check_transactionality":                           "unknown",
-	"test_execute.TestExecuteScript.test_execute_script_interrupted_close":                                "unknown",
 	"test_introspection.TestIntrospection.test_introspection_no_stmt_cache_01":                            "unknown",
 	"test_introspection.TestIntrospection.test_introspection_no_stmt_cache_02":                            "unknown",
 	"test_introspection.TestIntrospection.test_introspection_no_stmt_cache_03":                            "unknown",
@@ -96,12 +89,7 @@ var asyncpgIgnoreList22_1 = blocklist{
 	"test_listeners.TestLogListeners.test_log_listener_01":                                                "unknown",
 	"test_listeners.TestLogListeners.test_log_listener_02":                                                "unknown",
 	"test_listeners.TestLogListeners.test_log_listener_03":                                                "unknown",
-	"test_pool.TestPool.test_pool_handles_query_cancel_in_release":                                        "unknown",
-	"test_pool.TestPool.test_pool_handles_task_cancel_in_release":                                         "unknown",
-	"test_pool.TestPool.test_pool_init_race":                                                              "unknown",
-	"test_pool.TestPool.test_pool_init_run_until_complete":                                                "unknown",
 	"test_pool.TestPool.test_pool_remote_close":                                                           "unknown",
-	"test_prepare.TestPrepare.test_prepare_06_interrupted_close":                                          "unknown",
 	"test_prepare.TestPrepare.test_prepare_08_big_result":                                                 "unknown",
 	"test_prepare.TestPrepare.test_prepare_09_raise_error":                                                "unknown",
 	"test_prepare.TestPrepare.test_prepare_14_explain":                                                    "unknown",
@@ -115,19 +103,10 @@ var asyncpgIgnoreList22_1 = blocklist{
 	"test_record.TestRecord.test_record_subclass_02":                                                      "unknown",
 	"test_record.TestRecord.test_record_subclass_03":                                                      "unknown",
 	"test_record.TestRecord.test_record_subclass_04":                                                      "unknown",
-	"test_timeout.TestConnectionCommandTimeout.test_command_timeout_01":                                   "unknown",
-	"test_timeout.TestTimeout.test_timeout_04":                                                            "unknown",
-	"test_timeout.TestTimeout.test_timeout_06":                                                            "unknown",
 	"test_utils.TestUtils.test_mogrify_multiple":                                                          "unknown",
 	"test_utils.TestUtils.test_mogrify_simple":                                                            "unknown",
-	"test_pool.TestPool.test_pool_handles_transaction_exit_in_asyncgen_1":                                 "unknown",
-	"test_pool.TestPool.test_pool_handles_transaction_exit_in_asyncgen_2":                                 "unknown",
-	"test_listeners.TestListeners.test_dangling_listener_warns":                                           "unknown",
-	"test_pool.TestPool.test_pool_01":                                                                     "unknown",
-	"test_pool.TestPool.test_pool_init_and_use_race":                                                      "unknown",
 }
-
-var asyncpgIgnoreList21_2 = blocklist{
+var asyncpgBlocklist21_2 = blocklist{
 	"test_cache_invalidation.TestCacheInvalidation.test_prepare_cache_invalidation_in_pool":               "unknown",
 	"test_cache_invalidation.TestCacheInvalidation.test_prepare_cache_invalidation_in_transaction":        "unknown",
 	"test_cache_invalidation.TestCacheInvalidation.test_prepare_cache_invalidation_silent":                "unknown",
@@ -221,3 +200,6 @@ var asyncpgIgnoreList21_2 = blocklist{
 	"test_utils.TestUtils.test_mogrify_multiple":                                                          "unknown",
 	"test_utils.TestUtils.test_mogrify_simple":                                                            "unknown",
 }
+
+var asyncpgIgnoreList22_1 = asyncpgIgnoreList21_2
+var asyncpgIgnoreList21_2 = blocklist{}

